concurrent-modification/src/com/webonise/concurrentmodification/ConcurrentModificationException.java

```diff
@@ -0,0 +1,27 @@
+package com.webonise.concurrentmodification;
+
+import java.util.*;
+
+public class ConcurrentModificationException {
+
+    public static void main(String args[]) {
+        List<String> myList = new ArrayList<String>();
+
+        myList.add("first");
+        myList.add("second");
+        myList.add("third");
+        myList.add("fourth");
+        myList.add("fifth");
+
+        Iterator<String> it = myList.iterator();
+        while (it.hasNext()) {
+            String value = it.next();
+            System.out.println("List" + value);
+            if (value.equals("third")){
+                myList.remove(value);
+                //com.webonise.concurrentmodification.ConcurrentModificationException is thrown
```

Comment:
**com.webonise.concurrentmodification.ConcurrentModificationException** is yours class name , **java.util.ConcurrentModification** exception will be thrown.

This is a valid example why we have to avoid wild card imports as ` java.util` and `com.webonise `both have `ConcurrentModification `class.